docs/develop/update/redmine.rst

```diff
@@ -0,0 +1,91 @@
+Configure Redmine
```

Comment:
I think most of this file (Add basic configuration, Add plugins) is not necessary. We don't typically document single-instance services in detail. It's easy enough to read their SLS files. For multiple-instance services (PostgreSQL, Python apps, etc.) it's helpful to document.

This page can be more focused on the steps to upgrade Redmine and plugins.

Reply:
I've now made this change.